src/component_updater.py

```diff
@@ -0,0 +1,186 @@
+import logging
+import os
+from utils import io
+import tools
+from atmos_component import AtmosComponent, COMPONENT_YAML
+from github_provider import GitHubProvider
+from git_provider import GitProvider
+
+TERRAFORM_COMPONENTS_SUBDIR = 'components/terraform'
```

Comment:
This should be a parameter.

Reply:
Here it says that `components` is the required directory for infra repo. So I assumed that `components/terraform` is where customers will put their terraform stuff. 

https://atmos.tools/quick-start/configure-repository#common-directories-and-files

I'll parameterize it with default to `components/terraform`

Reply:
fixed